.github/actions/create-jira-issue/entrypoint.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+sprint_id=$(curl -D- -u ${JIRA_AUTH} \
+-X GET -H "Content-Type: application/json" https://scalar-labs.atlassian.net/rest/greenhopper/1.0/sprintquery/1  | sed -n '/^{/,$p' | jq '.sprints[] | select(.state=="ACTIVE")' | jq '.id')
```

Comment:
I think that `greenhopper` was deprecated.
https://community.atlassian.com/t5/Jira-questions/Can-we-use-JIRA-GREENHOPPER-REST-API-s-now/qaq-p/758344

Maybe we can use this api.
https://developer.atlassian.com/cloud/jira/software/rest/api-group-board/#api-agile-1-0-board-boardid-sprint-get

Reply:
Thank you. Fixed in 0f0a865